expansion/src/main/java/no/unit/nva/expansion/model/ExpandedResourceConversation.java

```diff
@@ -0,0 +1,66 @@
+package no.unit.nva.expansion.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import no.unit.nva.expansion.ResourceExpansionService;
+import no.unit.nva.expansion.WithOrganizationScope;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.publication.service.impl.ResourceConversation;
+import no.unit.nva.publication.storage.model.Message;
+import nva.commons.apigateway.exceptions.NotFoundException;
+
+import java.net.URI;
+import java.util.Set;
+
+@JsonTypeName("PublicationConversation")
+public class ExpandedResourceConversation extends ResourceConversation implements WithOrganizationScope, ExpandedDataEntry {
+
+    @JsonProperty("identifier")
+    private SortableIdentifier publicationIdentifier;
```

Comment:
I am not sure about calling this identifier. It is the publicationIdentifier.Is it possible to have publicationId here as well?

Reply:
I think there is a test that expects to find /identifier through the use of JsonPointer.